src/Accordion/Accordion.stories.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
-import { Accordion } from './Accordion'
+import { Accordion, AccordionProps } from './Accordion'
 
 export default {
   title: 'Accordion',
   component: Accordion,
 }
 
-const Template = (args) => (
+const Template = (args: AccordionProps) => (
```

Comment:
super nit, but shouldn't this be

```
props: AccordionProps
```

OR

```
args: accordionArgs
```
?

Reply:
I didn't think about changing anything in the stories, except when it was related to typing
The typing is related to props so it should definitely be `AccordionProps`
For the other one, I don't mind either way, I'll update it to `props` 👍